frontend_tests/casper_tests/04-subscriptions.js

```diff
@@ -12,13 +12,14 @@ casper.then(function () {
 });
 casper.waitForText('Subscribed', function () {
     casper.test.assertTextExists('Subscribed', 'Initial subscriptions loaded');
-    casper.fill('form#add_new_subscription', {stream_name: 'Waseemio'});
     casper.click('form#add_new_subscription input.btn');
 });
-casper.waitForText('Waseemio', function () {
-    casper.test.assertTextExists('Create stream Waseemio', 'Modal for specifying new stream users');
+casper.waitForText('Create stream', function () {
```

Comment:
Can we replace this with a `waitForSelector`?  Those are generally less fragile...